ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -110,7 +110,7 @@ export class ImportCommand extends IronfishCommand {
       const name = await CliUx.ux.prompt('Enter a new account name', {
         required: true,
       })
-      return { name, spendingKey }
+      return { name, spendingKey, version: 1 }
```

Comment:
Can we create a constant somewhere we can bump when version changes occur and reference it here?